lang/cy.js

```diff
@@ -7,12 +7,13 @@ export default {
 	"components.calendar.selected": "Wedi'i Ddewis.",
 	"components.calendar.show": "Dangos {month}",
 	"components.dialog.close": "Cau'r dialog hwn",
-	"components.dropdown.close": "Close",
-	"components.filter.loading": "Loading filters",
+	"components.dropdown.close": "Cau",
+	"components.filter.loading": "Wrthi’n llwytho hidlyddion",
 	"components.filter.filterCount": "{number, select, max {99+} all {I gyd} other {{number}}}",
 	"components.filter.filterCountDescription": "{number, plural, zero {Ni chymhwyswyd hidlwyr.} one {1 hidlydd wedi'i gymhwyso.} other {{number} hidlydd wedi'u cymhwyso.}}",
 	"components.filter.filters": "Hidlyddion",
-	"components.filter.noFilters": "No available filters",
+	"components.filter.noFilters": "Dim hidlyddion ar gael",
+	"components.filter.searchResults": "{number, plural, zero {No search results} one {1 search result} other {{number} search results}}",
```

Comment:
Seems like this didn't get translated properly? @svanherk

Reply:
I think this is just what happens when only some of our new terms get sent off for translation. Like what probably happened was `noFilters` was added first, translations went out, then `searchResults` was added. We'll get this one next time around.

Reply:
Yeah I believe this term was added this release (after 20.21.8 branching) so it just backfills it with English for now